airgun/views/ansible_role.py

```diff
@@ -0,0 +1,72 @@
+from widgetastic.widget import Checkbox
+from widgetastic.widget import Table
+from widgetastic.widget import Text
+from widgetastic_patternfly import BreadCrumb
+from widgetastic_patternfly4 import Button
+from widgetastic_patternfly4 import PatternflyTable
+
+from airgun.views.common import BaseLoggedInView
+from airgun.views.common import SearchableViewMixin
+from airgun.widgets import ActionsDropdown
+from airgun.widgets import Pagination
+from airgun.widgets import Pf4ConfirmationDialog
+
+
+class ImportPagination(Pagination):
+    PER_PAGE_BUTTON_DROPDOWN = ".//div[button[@id='pagination-options-menu-toggle-2']]"
+
+
+class AnsibleRolesView(BaseLoggedInView, SearchableViewMixin):
+    """Main Ansible Roles view. Prior to importing any roles, only the import_button
+    is present, without the search widget or table.
+    """
+
+    title = Text("//h1[contains(., text()='Ansible Roles')")
+    import_button = Text("//a[contains(@href, '/ansible_roles/import')]")
+    submit = Button('Submit')
+    total_imported_roles = Text("//span[contains(@class, 'pagination-pf-items-total')]")
```

Comment:
This is actually already available as `self.pagination.total_items`.

https://github.com/SatelliteQE/airgun/blob/master/airgun/widgets.py#L1463